libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager+Internal.h

```diff
@@ -32,6 +32,13 @@
 
 @class SFSDKAuthPreferences;
 
+extern NSString * _Nonnull const kSFSDKUserAccountManagerErrorDomain;
+
+typedef NS_ENUM(NSUInteger, SFSDKUserAccountManagerErrorCode) {
```

Comment:
@trooper2013  this is where I'm thinking the other SFUserAccountManagerErrorDomain codes could move to when it's ready, the others were 10001-10004, so could drop them in or renumber them all later